charts/secrets-injector/templates/deployment.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.injector.applicationName }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ .Values.injector.applicationName }}
+  annotations:
+    helm.sh/hook: pre-install
+    helm.sh/hook-weight: "1"
+spec:
+  replicas: {{ .Values.injector.replicas }}
+  selector:
+    matchLabels:
+      app: {{ .Values.injector.applicationName }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.injector.applicationName }}
+    spec:
+      serviceAccountName: {{ .Values.injector.applicationName }}
+      containers:
+        - name: {{ .Values.injector.applicationName }}
+          image: {{ .Values.injector.imageRepository }}:{{ .Chart.AppVersion }}
```

Comment:
This gives me the impression that the user can't change the version of the image. We should enable the users to do that.

I see that for Connect we have something like this:
```yaml
image: {{ .Values.connect.api.imageRepository }}:{{ tpl .Values.connect.version . }}
```

Maybe something similar works here as well?

Reply:
Yep, that works! Updated.